roles/azure_load_balancer_with_public_ip/README.md

```diff
@@ -43,6 +43,8 @@ Role Variables
     - **protocol**: IP protocol for the rule. Valid choices are: 'Tcp', 'Udp', 'All'.
   - **sku**: Load balancer SKU. Valid choices are: 'Basic', 'Standard'. Will also be applied to the public ip generated for the load balancer.
   - **tags**: Dictionary of string:string pairs to assign as metadata to the load balancer.
+  - **azure_load_balancer_with_public_ip_delete_resource_group**: Relevant for **create** operation. Delete the resource group.
```

Comment:
It might be better to add a description of this variable after the description of the azure_load_balancer_with_public_ip_operation variable. Additionally, specify the cases when the resource group is created and deleted by the role.

Reply:
I think this is fine to have it in the end as this variable doesn't related directly to the role
We will put it always in the end, in all roles variables lists

Reply:
Done